backend/Dockerfile.dev

```diff
@@ -0,0 +1,7 @@
+FROM node:14.11.0 AS api-development
+RUN mkdir /srv/backend
+WORKDIR /srv/backend
+RUN mkdir -p node_modules
+COPY package.json yarn.lock ./
+RUN yarn install --pure-lockfile
+COPY . .
```

Comment:
please, add an empty line at the end of the file
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline